fcrepo-kernel-api/src/test/java/org/fcrepo/kernel/api/services/functions/ConfigurableHierarchicalSupplierTest.java

```diff
@@ -18,14 +21,14 @@
  *
  * @author rdfloyd
  */
-public class ConfigurableHierarchicalSupplierTest {
-
-    private FedoraPropsConfig fedoraPropsConfig;
-
-    @Before
-    public void setUp() {
-        fedoraPropsConfig = new FedoraPropsConfig();
-    }
+@SpringBootTest
+@SpringBootConfiguration
+@RunWith(SpringJUnit4ClassRunner.class)
+@TestPropertySource(properties = {
+    "fcrepo.pid.minter.length=2",
+    "fcrepo.pid.minter.count=4"
+})
```

Comment:
Assuming this sets the properties for all the tests, then the first two tests should be failing now. I'm having trouble pulling a ref to the merge of this PR. But reading these changes it appears that originally we set the length and count on one test, so setting it for all 3 should cause a failure.